### ISSUE TITLE
Radlistview disable scroll

### ISSUE THREAD

**Issue by aminebizid** [telerik/nativescript-ui-feedback#909]
We have many listviews inside a scroll view. 
The scrolling works great on ios but is captured by listviews on Android.
The height of our listviews is exactly the same of the height of contained items. 
Any suggestion how to disable the scrolling inside a list view?

**Comment by aminebizid** [comment]
@tsonevn Thank you for your answer.
We are mixing grouping and swipe gestures inside the Radlistview and this as you may know doesn't work. This is why we created a RadListView per group.

**Comment by tsonevn** [comment]
Hi @aminebizid,
You can group the items in the RadListView component. Check out the example here:
https://github.com/telerik/nativescript-ui-samples/tree/master/listview/app/examples/grouping
https://github.com/telerik/nativescript-ui-samples/tree/master/listview/app/examples/grouping-multiple-templates

**Comment by aminebizid** [comment]
Yes I know but please try to add swiping on a grouped list and gonna see
the strange behavior

Le lun. 5 nov. 2018 à 12:35, Nikolay Tsonev <notifications@github.com> a
écrit :
